articles/migrations/index.md

```diff
@@ -208,7 +208,7 @@ You are affected by the change only if you are using the Identity Provider acces
 For more information on how to obtain an access token, see: [Call an Identity Provider API](/what-to-do-once-the-user-is-logged-in/calling-an-external-idp-api) and [Identity Provider Access Token](/tokens/idp).
 
 ::: note
-If your account was created after the change, this update will be applied automatically.
+If your tenant was created after the change, this update will be applied automatically.
```

Comment:
Do you know this for sure? Are migrations applied based on when a tenant was created and not the account? (I'm not saying it's wrong, I'm just not sure)

Reply:
Yeah, I just reread the [notes](https://github.com/auth0/auth0-server/issues/2283). From what I understand, it has always been tenant, never account.